src/dataflow/src/decode/mod.rs

```diff
@@ -393,6 +399,74 @@ where
     })
 }
 
+fn decode_cdcv2<G: Scope<Timestamp = Timestamp>>(
+    stream: &Stream<G, SourceOutput<Vec<u8>, Vec<u8>>>,
+    schema: &str,
+    registry: Option<ccsr::ClientConfig>,
+) -> (Stream<G, (Row, Timestamp, Diff)>, Option<Box<dyn Any>>) {
+    let mut resolver = ConfluentAvroResolver::new(schema, registry).unwrap(); // We will have already checked validity of the schema by now, so this can't fail.
+    let channel = Rc::new(RefCell::new(VecDeque::new()));
+    let activator: Rc<RefCell<Option<SyncActivator>>> = Rc::new(RefCell::new(None));
+    let mut vector = Vec::new();
+    let stream = stream.unary::<(), _, _, _>(
```

Comment:
Do you want a `sink` here, or something without an output? I presume the `()` is to specify the output type because you don't actually use it?